src/components/Documents/DocumentTableRow.jsx

```diff
@@ -64,6 +65,11 @@ const DocumentTableRow = ({ document }) => {
           <FileOpenIcon />
         </IconButton>
       </StyledTableCell>
```

Comment:
Not too urgent of a fix in this PR, but was wondering when we'll be phasing out the styled-components here with MUI components.

Reply:
I don't think we need to worry about this here. You can put up another issue about it. Would be a good first issue.

Reply:
I believe's there's already a related issue set in Issue #219.